internal/app/api/module.go

```diff
@@ -76,18 +76,19 @@ func HTTPModule() fx.Option {
 func httpRouter(db *mongo.Database, client *mongo.Client, handlers []connectorHandler) (*mux.Router, error) {
 	rootMux := mux.NewRouter()
 
-	if viper.GetBool(otelTracesFlag) {
-		rootMux.Use(otelmux.Middleware(serviceName))
-	}
-
 	rootMux.Use(recoveryHandler(httpRecoveryFunc))
 	rootMux.Use(httpCorsHandler())
 	rootMux.Use(httpServeFunc)
 
+	// Deprecated
 	rootMux.Path("/_health").Handler(healthHandler(client))
```

Comment:
We defined than health check must be /_healthcheck.
Keep for backward compatibility.